packages/superset-ui-preset-chart-xy/src/encodeable/AxisAgent.ts

```diff
@@ -54,8 +54,21 @@ export default class AxisAgent<Def extends ChannelDef<Output>, Output extends Va
     return this.format || this.channelEncoder.formatValue;
   }
 
+  hasTitle() {
+    return this.getTitle() !== undefined;
+  }
+
   getTitle() {
-    return this.config.title || this.channelEncoder.getTitle();
+    if (typeof this.config.title === 'undefined' || this.config.title === true) {
```

Comment:
Since it's an object property I *think* you can safely do `this.config.title === undefined`?